LibationWinForms/GridEntry.cs

```diff
@@ -26,12 +29,15 @@ internal class GridEntry : AsyncNotifyPropertyChanged, IMemberComparable
 
 		private Book Book => LibraryBook.Book;
 		private Image _cover;
+		private Action _refilter;
```

Comment:
Personal preference: mark this as {get;} or readonly so it can't accidentally be overwritten elsewhere